src/views/console/DeleteProfileView.ts

```diff
@@ -0,0 +1,34 @@
+import { createInterface } from "readline";
+import { DeleteProfileController } from "../../controllers/DeleteProfileController";
+
+export class DeleteProfileView {
+  private readonly controller: DeleteProfileController;
+
+  constructor() {
+    this.controller = new DeleteProfileController();
+  }
+
+  private async readString(msg: string): Promise<string> {
```

Comment:
Merge main into this branch, and extends ConsoleView on this class. New protected console is ready!